{"name": "test_validate_place[18-Unter den Linden 18-Berlin-NaN-NaN]", "status": "broken", "statusDetails": {"message": "pydantic.error_wrappers.ValidationError: 1 validation error for Place\naddress -> house_number\n  field required (type=value_error.missing)", "trace": "tests\\address_to_coordinate_test.py:30: in test_validate_place\n    assert Place.parse_obj(request_address(place_to_parse).json()[0]), 'address not valid'\npydantic\\main.py:526: in pydantic.main.BaseModel.parse_obj\n    ???\npydantic\\main.py:341: in pydantic.main.BaseModel.__init__\n    ???\nE   pydantic.error_wrappers.ValidationError: 1 validation error for Place\nE   address -> house_number\nE     field required (type=value_error.missing)"}, "parameters": [{"name": "id", "value": "'18'"}, {"name": "street", "value": "'Unter den Linden 18'"}, {"name": "city", "value": "'Berlin'"}, {"name": "lat", "value": "'NaN'"}, {"name": "lon", "value": "'NaN'"}], "start": 1686543475472, "stop": 1686543477507, "uuid": "202b732b-3f8d-476f-a5e3-4cd54b607fd6", "historyId": "c211550a0c0ff75d0112ac9f4ef5591d", "testCaseId": "c417ba1583178e81f87b166ea3c9d081", "fullName": "tests.address_to_coordinate_test.TestGettingDataByAddress#test_validate_place", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "address_to_coordinate_test"}, {"name": "subSuite", "value": "TestGettingDataByAddress"}, {"name": "host", "value": "DESKTOP-20A052D"}, {"name": "thread", "value": "4348-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.address_to_coordinate_test"}]}